packages/components/ng-log-live-tail/src/template.html

```diff
@@ -60,23 +60,29 @@
             ></div>
 
             <div data-ng-repeat="log in $ctrl.logs track by $index">
-                <div>
-                    <span
-                        data-ng-bind-html="log['formattedTimestamp_highlighted']"
-                        class="log-timestamp"
-                    ></span>
+                <span
+                    data-ng-bind-html="log['formattedDate_highlighted']"
+                    class="log-date"
+                ></span>
+                <span
+                    data-ng-bind-html="log['formattedTime_highlighted']"
+                    class="log-time"
+                ></span>
+                <span class="separator">|</span>
```

Comment:
**niptick**: could be done in css with `.separator::after {content : '|'}` to avoid repeating it in the html view